examples/xo_android_client/app/src/main/java/io/bitwise/sawtooth_xo/MainActivity.kt

```diff
@@ -1,21 +1,69 @@
 package io.bitwise.sawtooth_xo
 
+import android.arch.lifecycle.ViewModelProviders
 import android.content.Intent
 import android.support.v7.app.AppCompatActivity
 import android.os.Bundle
 import android.support.design.widget.FloatingActionButton
+import android.view.Menu
+import android.view.MenuItem
+import android.view.View
+import io.bitwise.sawtooth_xo.models.Game
+import android.support.design.widget.TabLayout
+import android.support.v4.view.ViewPager
+import io.bitwise.sawtooth_xo.adapters.PagerAdapter
+import io.bitwise.sawtooth_xo.viewmodels.GameViewModel
 
-class MainActivity : AppCompatActivity() {
+
+class MainActivity : AppCompatActivity(),  GameListFragment.OnListFragmentInteractionListener{
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+
         setContentView(R.layout.activity_main)
+        setSupportActionBar(findViewById(R.id.list_view_toolbar))
+        val viewPager = findViewById<ViewPager>(R.id.pager)
+        setupViewPager(viewPager)
+        val tabs = findViewById<View>(R.id.tabs) as TabLayout
+        tabs.setupWithViewPager(viewPager)
+
 
         val fab: FloatingActionButton = findViewById(R.id.newGameFloatingButton)
-        fab.setOnClickListener { view ->
+        fab.setOnClickListener { _ ->
             val intent = Intent(this, GameBoardActivity::class.java)
             startActivity(intent)
         }
 
     }
+
+    private fun setupViewPager(viewPager: ViewPager) {
+        val adapter = PagerAdapter(supportFragmentManager)
+        adapter.addFragment(GameListFragment(), "Play")
+        adapter.addFragment(GameListFragment(), "Observe")
```

Comment:
During the design discussion we talked about the tab being called "Spectate". I am in favor of "Spectate" as it has a more game/sport related context.

Reply:
I would leave these tabs out for now, until they're functional.